### ISSUE TITLE
Plugin exited in pipeline

### ISSUE THREAD

**Issue by pdmthorsrud** [Skatteetaten/terraform-nomad-trino#124]
## Current behaviour
Pipeline throws an error 
https://github.com/Skatteetaten/terraform-nomad-trino/pull/122/checks?check_run_id=2692198092
https://github.com/Skatteetaten/terraform-nomad-trino/pull/122/checks?check_run_id=2692198092

## Expected behaviour
Pipeline runs successfully

## How to reproduce?
Run github actions in any pull request

## Checklist (after created issue)
- [x] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

**Comment by fredrikhgrelland** [comment]
This looks to be a timeout.
@lmjelstad I believe we need to handle nomad ACLs here properly.

**Comment by fredrikhgrelland** [comment]
related https://github.com/Skatteetaten/terraform-nomad-trino/issues/119

**Comment by lmjelstad** [comment]
@fredrikhgrelland I agree that it looks like a timeout, deploy of nomad job stops after 5m and is not successfully deployed https://github.com/Skatteetaten/terraform-nomad-trino/pull/122/checks?check_run_id=2692198092#step:11:3118

FYI, the `pipeline exited` message also appears on successful pipeline, so I'm not sure if this is the error https://github.com/Skatteetaten/terraform-nomad-trino/pull/122/checks?check_run_id=2692198062#step:11:13215 

I believe I have also seen the error testing locally, where event logs in trino job in nomad said that the pull of trino docker image failed due to `context canceled` indicating a possible timeout during the pull

**Comment by pdmthorsrud** [comment]
> @fredrikhgrelland I agree that it looks like a timeout, deploy of nomad job stops after 5m and is not successfully deployed https://github.com/Skatteetaten/terraform-nomad-trino/pull/122/checks?check_run_id=2692198092#step:11:3118
> 
> FYI, the `pipeline exited` message also appears on successful pipeline, so I'm not sure if this is the error https://github.com/Skatteetaten/terraform-nomad-trino/pull/122/checks?check_run_id=2692198062#step:11:13215
> 
> I believe I have also seen the error testing locally, where event logs in trino job in nomad said that the pull of trino docker image failed due to `context canceled` indicating a possible timeout during the pull

Good point on the `Plugin exited` part, that does indeed sound like something that is terminated (because of a timeout like you said), so it isn't necessarily the root of the error, I agree! 

There is quite a while since I've seen any `context canceled` locally, if that is of any help.

**Comment by lmjelstad** [comment]
> There is quite a while since I've seen any `context canceled` locally, if that is of any help.

I experienced problems with `context canceled` locally last week
